packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -1,28 +1,28 @@
-import net from 'node:net';
-import fs from 'fs-extra';
-import os from 'node:os';
-import jwt from 'jsonwebtoken';
+import { createServer } from 'node:net';
```

Comment:
whats the advantage of these imports, e.g. on os methods I find it pretty useful if its eg `os.networkInterfaces`, why should we refactor all of this

Reply:
- Better tree-shaking: If you're using a bundler (e.g., Webpack), only the existsSync function is included in your final bundle (if the environment supports proper ES module tree-shaking). __(Not our case, but to be consequent)__ with fron end
- Smaller imports: Reduces memory overhead and improves clarity if you're only using specific functions.

Reply:
And in our case at least we can avoid the mix of argument `path` (used ca 54 times) and library `node:path`

Reply:
> Advantages:
> 
> * Better tree-shaking: If you're using a bundler (e.g., Webpack), only the existsSync function is included in your final bundle (if the environment supports proper ES module tree-shaking). **(Not our case, but to be consequent with fron end)**
> * Smaller imports: Reduces memory overhead and improves clarity if you're only using specific functions.
> 
> Disadvantages:
> 
> * No access to other methods in the module unless explicitly imported.
> * Slightly less clear in larger codebases where existsSync might come from if you don’t refer back to the import statement.
> 
> So I use such a calls in Front-End everywhere to make a smaller size of compiled code

I have also read about in the past as I was curious and my conslusion was for backend code (and for me personally also for frontend), Maintainability is a lot more important then the potential of gaining a single ms and in Backend mostly none due to 1,000 optimized imports. Yes for Frontend I understand it if it really works out in the end.

For here,  I really like to decide this on per-use if its a 5 lines file it might be more applicable to make ES6 style import. Lets focus on the real benefits and not waste time on that stuff, I would really appreciate a environement where we just touch the code needed. Then we get rid of a lot of these conversations which we both don't like to waste our time with 👍

Reply:
adapter.ts was changed anyway

Reply:
I changed the import back. But let the `path` in new notation to avoid the mix with `path`variables